gradle-plugins/publish-to-maven-local.sh

```diff
@@ -1,2 +1,10 @@
-cd ./gradle-plugins/react && ./gradlew build || exit 1
-./gradlew :brownfield:publishToMavenLocal
\ No newline at end of file
+#!/bin/bash
+
+set -e
+
+cd ./gradle-plugins/react
+./gradlew clean
+./gradlew build
+./gradlew :brownfield:publishMavenLocalPublicationToMavenLocalRepository
+./gradlew :brownfield:signMavenLocalPublication
```

Comment:
Do we need to call this manually? In my previous attempt it was enough to call `./gradlew publishToMavenLocal` to have everything working.